lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +277,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: Content-specific discussion topic ID
```

Comment:
@nasthagiri to answer your question, i'm pretty sure these are all equivalent: `topic_id` / `discussion_id` / `commentable_id`  (roughly mapping to terminology of discussion API, django comment client, comments service)

naming things...